### ISSUE TITLE
Standardize error handling

### ISSUE THREAD

**Issue by bhavaanikakarla** [covidFYI/frontend#20]
1. Create Error Logs so that all bugs can be tracked. Integrate something like (https://airbrake.io/) to track the errors and log errors and manage production user errors easily.

2. Use backend error codes and centralize the error messages by creating Error Types on front end that would have fields like - Code, Name and Description for different errors.

Use these error message types for all errors. For every new type of error, add an entry to the Error Type.

**Comment by tejasa97** [comment]
Yes,  had deferred on this because I understood that error handling would be required only if there was any data we were taking from the user, i.e : POST requests, etc.
But right now, we only have faulty API requests being made from the frontend like /state/null, /state/undefined, which is primarily because of bugs in the frontend. do you think we should log/standardize these too? @bhavaanikakarla @yogesh-hackx @manan2002
